core/hub/sync.go

```diff
@@ -47,7 +47,8 @@ func QueryHub(ctx context.Context, node *core.IpfsNode, mode string) ([]*hubpb.H
 	err = grpc.HubQueryClient(config.Services.HubDomain).WithContext(ctx, func(ctx context.Context,
 		client hubpb.HubQueryServiceClient) error {
 		resp, err = client.GetHosts(ctx, &hubpb.HostsReq{
-			Id: node.Identity.Pretty(),
+			Id:       node.Identity.Pretty(),
+			RespSize: 30,
```

Comment:
The goal was to grab the default (100) from hub, why set 30 here?

Reply:
i can unset, it only for early stages when hub returning hosts not consistent with set hosts

Reply:
no I can't I tested if I setting to 100, the order returning from hub is not what I am setting here